src/components/Autocomplete/utils/helpers.ts

```diff
@@ -0,0 +1,9 @@
+export function isIosDevice() {
```

Comment:
Move this into the utils directory under the src directory. Rename the file with something more meaningful.